openedx_learning/apps/authoring/backup_restore/toml.py

```diff
@@ -6,19 +6,30 @@
 from typing import Any, Dict
 
 import tomlkit
+from django.contrib.auth.models import User as UserType  # pylint: disable=imported-auth-user
 
 from openedx_learning.apps.authoring.collections.models import Collection
 from openedx_learning.apps.authoring.publishing import api as publishing_api
 from openedx_learning.apps.authoring.publishing.models import PublishableEntity, PublishableEntityVersion
 from openedx_learning.apps.authoring.publishing.models.learning_package import LearningPackage
 
 
-def toml_learning_package(learning_package: LearningPackage, timestamp: datetime) -> str:
+def toml_learning_package(
+        learning_package: LearningPackage,
+        timestamp: datetime,
+        format_version: int = 1,
+        user: UserType | None = None,
+        origin_server: str | None = None
+        ) -> str:
     """
     Create a TOML representation of the learning package.
 
     The resulting content looks like:
-        # Datetime of the export: 2025-09-03 12:50:59.573253
+        [meta]
+        format_version = 1
+        created_by = "dormsbee"
+        created_at = 2025-09-03T17:50:59.536190Z
+        origin_server = "http://cms.test"
```

Comment:
```suggestion
        origin_server = "cms.test"
```

Nit: The value from CMS_BASE isn't going to include "http://".